tests/unit/test_missing_init_files.py

```diff
@@ -34,6 +34,10 @@ def _find_missing():  # pragma: nocover
         if rel_root.startswith("bdd"):
             continue
 
+        # Ignore the cookiecutter's pytest_plugins dir.
+        if rel_root == "pytest_plugins":
+            continue
```

Comment:
The `pytest_plugins` directory that the cookiecutter creates doesn't need to and doesn't contain an `__init__.py`. If the cookiecutter did put an `__init__.py` in this directory then we'd need logic in the cookicutter to say "if there are no pytest plugins in the directory then don't put the `__init__.py` either" which would be possible but awkward. Or we'd have to accept projects getting a `pytest_plugins` directory containing nothing but an empty `__init__.py`.

This does mean that when you run `make lint` in a project pylint isn't linting the cookiecutter's pytest plugins but that's probably okay.